doc/manual/strings.rst

```diff
@@ -768,7 +768,7 @@ For example::
 Numbered capture groups can also be referenced as ``\g<n>`` for disambiguation,
 as in::
 
-    julia> replace("a", r".", "\g<0>1")
+    julia> replace("a", r".", s"\g<0>1")
     julia> a1
```

Comment:
the prompt shouldn't be here, and this should be returning a string. we should make all the code blocks in this file into doctests if possible.